drools-benchmarks/src/main/java/org/drools/benchmarks/turtle/runtime/KBaseCreationFromDTable2Benchmark.java

```diff
@@ -0,0 +1,53 @@
+package org.drools.benchmarks.turtle.runtime;
+
+import org.drools.benchmarks.common.util.RuntimeUtil;
+import org.drools.benchmarks.turtle.runtime.generator.GeneratorConfiguration;
+import org.drools.benchmarks.turtle.runtime.generator.KBaseCreationFromDTable2Generator;
+import org.kie.api.runtime.KieSession;
+import org.kie.api.runtime.conf.ClockTypeOption;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Level;
+import org.openjdk.jmh.annotations.Setup;
+
+public class KBaseCreationFromDTable2Benchmark extends AbstractSimpleRuntimeBenchmark {
+
+    @Setup
+    public void addResources() {
+        addClassPathResource("kbase-creation/dtable2-kbase-creation.xls");
+    }
+
+    @Setup(Level.Iteration)
+    public void createKieSession() {
+        ksession = RuntimeUtil.createKieSession(kieBase, ClockTypeOption.get("pseudo"));
```

Comment:
Same comment as in the previous benchmark class. Please remove this and insertFactsAndFireAllRules() method.